### PR TITLE
Pull Request - express-rest-service

### PR DESCRIPTION
**Pull request by 96tm**
1. Task
https://github.com/rolling-scopes-school/basic-nodejs-2021Q2#task-2-express-rest-service
2. Done / deadline
16.05.21 / 16.05.21
3. How to check 
```sh
$ git clone https://github.com/96tm/nodejs-course-template.git
$ cd nodejs-course-template
$ git switch express-rest-service
$ npm i
$ npm run lint
$ npm start
```
Then in a different terminal
```sh
$ npm test
```
4. Self-check
200
5. Screenshot
screenshot
6. Note
I 'forked' the template at about 06.05.21 and completed the task at 16.05.21. But then I learned that files package.json and .eslintrc.json were changed and directory .github was added in the upstream, so I just copied them from the rsschool template (also copied package-lock.json, not sure if I had to). I don't think there's going to be any problem about that, but it seems it's better to inform you just in case.

### CODE REVIEW

File: src/resources/boards/board.router.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  res.json(tasks);
+});
+
+router.route('/:boardId/tasks/:taskId').get(async (req, res) => {
+  const { boardId, taskId } = req.params;
+  const task = await tasksService.getByBoardAndTaskId(boardId, taskId);
+  if (task) {
+    res.json(task);
+  } else {
+    res.status(404).json('Task not found');
```
Comment (spitaI) [inline src/resources/boards/board.router.js:34]: It's a bit strange that you're passing a simple string into **.json()** method :)

File: src/resources/boards/board.router.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  }
+});
+
+router.route('/:boardId/tasks/:taskId').put(async (req, res) => {
+  const { boardId, taskId } = req.params;
+  const { title, order, description, userId, columnId } = req.body;
+  const task = await tasksService.editByBoardAndTaskId({
```
Comment (spitaI) [inline src/resources/boards/board.router.js:41]: You could just do
```javascript
const task = await tasksService.editByBoardAndTaskId({ ...req.body, boardId, taskId });
```

Comment (96tm) [reply src/resources/boards/board.router.js:41]: Thanks, I'll change it when we add a database.

File: src/resources/boards/board.router.js (hunk 1 above)
Comment (96tm) [reply src/resources/boards/board.router.js:34]: Right, should have written ```json.({'message':  'Task not found'})```.

**Pull request merged**